WordPress/Classes/ViewRelated/Post/Revisions/Views/RevisionDiffsBrowserViewController.swift

```diff
@@ -0,0 +1,103 @@
+import Gridicons
+
+struct RevisionBrowserState {
+    let revisions: [Revision]
+    var currentIndex: Int
+
+    func currentRevision() -> Revision {
+        return revisions[currentIndex]
+    }
+    mutating func decreaseIndex() {
+        currentIndex = max(currentIndex - 1, 0)
+    }
+    mutating func increaseIndex() {
+        currentIndex = min(currentIndex + 1, revisions.count)
+    }
+}
+
+class RevisionDiffsBrowserViewController: UIViewController {
+    var revisionState: RevisionBrowserState?
+    var diffVC: RevisionDiffViewController?
+    var operationVC: RevisionOperationViewController?
+    @IBOutlet var revisionTitle: UILabel!
+    @IBOutlet var previousButton: UIButton!
+    @IBOutlet var nextButton: UIButton!
+
+
+    private lazy var doneBarButtonItem: UIBarButtonItem = {
+        let doneItem = UIBarButtonItem(barButtonSystemItem: .done, target: nil, action: nil)
+        doneItem.on() { [weak self] _ in
+            self?.dismiss(animated: true)
+        }
+        doneItem.title = "Done"
```

Comment:
Should this be localized?

Reply:
Oh darn, ya, that was meant to be temporary 😬